pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -74,6 +74,10 @@ def world_size(self) -> int:
     def root_device(self) -> torch.device:
         return xm.xla_device()
 
+    @property
+    def is_local_zero(self) -> bool:
+        return self.local_rank == 0
+
```

Comment:
does this need to be exposed more broadly? what other callsites rely on it?

Reply:
Not really, removed!